erpnext/patches/v13_0/update_delivery_note_status.py

```diff
@@ -0,0 +1,25 @@
+import frappe
+
+def execute():
+	stops = frappe.get_all("Delivery Stop", fields=["visited", "sales_invoice", "delivery_note", "parent"])
+	for stop in stops:
+		if stop.delivery_note:
+			if stop.visited and stop.sales_invoice:
+				si_status = frappe.db.get_value("Sales Invoice", stop.sales_invoice, "status")
+				if si_status == "Paid":
+					frappe.db.set_value("Delivery Note", stop.delivery_note, {
+						"delivered": 1,
```

Comment:
The patch is failing trying to set this field, reload the doctypes before running this one